packages/office-addin-dev-settings/src/sideload.ts

```diff
@@ -210,6 +213,20 @@ function isSideloadingSupportedForWebHost(app: OfficeApp): boolean {
   return false;
 }
 
+async function getOutlookVersion(): Promise<string | undefined> {
+  try {
+    const OutlookInstallPathVersionRegistryKey = `HKEY_LOCAL_MACHINE\\SOFTWARE\\Microsoft\\Office\\ClickToRun\\Configuration`;
+    const key = new registry.RegistryKey(`${OutlookInstallPathVersionRegistryKey}`);
+    const outlookInstallVersion: string | undefined = await registry.getStringValue(key, "ClientVersionToReport");
+    const outlookSmallerVersion = outlookInstallVersion?.split(`.`, 3).join(`.`);
```

Comment:
Why do you split and join here?

Reply:
Because the version that gets returned is in the format:  3 "a.b.c.d", but the API only supports version in the format: "a.b.c"
Given that we only care about the first 3 steps, I do this split join step.